.github/workflows/pr-test.yml

```
@@ -61,4 +61,6 @@ jobs:
           format: 'table'
           severity: 'CRITICAL'
           exit-code: '1'
-          ignore-unfixed: true # Won't fail if there isn't a fix to the CVE
\ No newline at end of file
+          ignore-unfixed: true # Won't fail if there isn't a fix to the CVE
+        env:
+          TRIVY_IGNORE: .trivyignore
```

Based on the docs you should be able to 

```suggestion
          ignore-unfixed: true # Won't fail if there isn't a fix to the CVE
          trivyignores: .trivyignore
```